sormas-backend/src/main/java/de/symeda/sormas/backend/caze/caseimport/CaseImportFacadeEjb.java

```diff
@@ -654,7 +654,11 @@ private boolean isPersonSimilarToExisting(PersonDto referencePerson) {
 			.passportNumber(referencePerson.getPassportNumber())
 			.nationalHealthId(referencePerson.getNationalHealthId());
 
-		return personFacade.getMatchingNameDtos(userFacade.getCurrentUser().toReference(), criteria).stream().findAny().isPresent();
+//		boolean present = personFacade.getMatchingNameDtos(userFacade.getCurrentUser().toReference(), criteria).stream().findAny().isPresent();
+
+		boolean present = personFacade.checkMatchingNameDtos(userFacade.getCurrentUser().toReference(), criteria);
```

Comment:
No need for a variable declaration here, just return the result immediately

Reply:
removed